src/utils/violet/styled.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components/macro'
+
+const EmbeddedAuthWrapper = styled.div`
```

Comment:
It looks like EmbeddedAuth is never used without this wrapped. Can we make this implicit to the EmbeddedAuth component that gets exported by the sdk instead? Or is there a reason why it's better to do it this way?

Reply:
+1 
From an integration perspective, 1 import is simpler than 2. We might want to add some extra styling on Mauve's side to fit the rest of the app with a wrapper but this CSS here looks pretty generic so may as well follow your suggestion...